src/main/java/org/folio/edge/patron/PatronHandler.java

```diff
@@ -65,14 +70,13 @@ protected void handleCommon(RoutingContext ctx, String[] requiredParams, String[
 
     String offsetParam = ctx.request().getParam(PARAM_OFFSET);
     if (isRequestIntegerParamWrong(offsetParam, true)) {
-      badRequest(ctx, String.format("'offset' parameter is incorrect. parameter value {%s} is not valid: must be "
-        + "an integer, greater than or equal to 0", offsetParam));
+      badRequest(ctx, String.format(String.format(WRONG_INTEGER_PARAM_MESSAGE, OFFSET, offsetParam), offsetParam));
       return;
     }
 
     String limitParam = ctx.request().getParam(PARAM_LIMIT); {
-      if (isRequestIntegerParamWrong(limitParam, false)) {
-        badRequest(ctx, String.format("'limit' parameter is incorrect. parameter value {%s} is not valid: must be an integer", limitParam));
+      if (isRequestIntegerParamWrong(limitParam, true)) {
```

Comment:
second param in isRequestIntegerParamWrong() always true. Please, reformat the logic

Reply:
Changed

Reply:
Changed